tests/actions.go

```diff
@@ -516,11 +520,6 @@ func (oa *operatorActions) DeployOperatorOrDie(info *OperatorConfig) {
 func (oa *operatorActions) CleanOperator(info *OperatorConfig) error {
 	glog.Infof("cleaning tidb-operator %s", info.ReleaseName)
 
-	err := oa.CleanWebHookAndService(info.WebhookConfigName)
-	if err != nil {
-		return err
-	}
-
```

Comment:
webhook configuration is not deployed in `DeployOperator` anymore, no need to clean here. I remove this because this requires us to configure webhook names even if we don't use webhook.